index.html

```diff
@@ -32,5 +32,159 @@ <h3>LET'S CONNECT</h3>
+                        or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li class="language" id="html">HTML</li>
+                        <li class="language" id="css">CSS</li>
+                        <li class="language" id="javascript">Javascript</li>
+                    </ul>
+                    <button class="button">see project</button>
+                </div>
+            </div>
+            <div class="projects">
+                <img class="snapshots" id="snap3" src="snap2.svg">
+                <div class="info_block">
+                    <div class="projo">
+                        <h4 class="project_name">Tonic</h4>
+                        <ul class="deatils">
+                            <li class="detail">CANOPY</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">Back End Dev</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">2023</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li class="language" id="html">HTML</li>
+                        <li class="language" id="css">CSS</li>
+                        <li class="language" id="javascript">Javascript</li>
+                    </ul>
+                    <button class="button">see project</button>
+                </div>
+            </div>
+            <div class="projects">
+                <img class="snapshots" id="snap4" src="snap3.svg">
+                <div class="info_block">
+                    <div class="projo">
+                        <h4 class="project_name">Multi-Post Stories</h4>
+                        <ul class="deatils">
+                            <li class="detail">CANOPY</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">Back End Dev</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">2023</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li class="language" id="html">HTML</li>
+                        <li class="language" id="css">CSS</li>
+                        <li class="language" id="javascript">Javascript</li>
+                    </ul>
+                    <button class="button">see project</button>
+                </div>
+            </div>
```

Comment:
- [ ] I would kindly suggest you to work more on all your `see project button`as its not align like the original figma template  follow the button styling: Active, Hover and Pressed (see design)
- [ ] your design
![Screenshot 2023-06-03 041755](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/495d1730-cf51-402b-84bf-3612ed0e51a0)

- [ ] figma design
![Screenshot 2023-06-03 041713](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/e8dde6e4-685d-432c-ba3a-432c9dda86bb)